backend/services/interfaces/testService.ts

```diff
@@ -0,0 +1,31 @@
+import { Question } from "../../models/test.model";
+import { UserDTO } from "../../types";
+
+export type TestResponseDTO = {
+  /** the unique identifier of the response */
+  id: string;
+  /** the name of the student */
+  name: string;
+  /** the duration of the test */
+  duration: number;
+  /** the UserDTO for the admin */
+  admin: UserDTO;
+  /** an array of questions on the test */
+  questions: Question[];
+  /** the grade of the student */
+  grade: number;
+};
+
+export type CreateTestRequestDTO = Omit<TestResponseDTO, "id" | "admin"> & {
+  admin: string;
```

Comment:
is this the `id` for the admin in place of the appropriate `UserDTO`? If so, can we leave a comment documenting this? Thanks!